package.json

```diff
@@ -4,47 +4,70 @@
   "author": "Tobias Koppers @sokra",
   "description": "worker loader module for webpack",
   "scripts": {
-    "test": "mocha",
-    "posttest": "eslint ."
-  },
-  "eslintConfig": {
-    "extends": "webpack",
-    "rules": {
-      "linebreak-style": 0,
-      "comma-dangle": [
-        "error",
-        {
-          "arrays": "always-multiline",
-          "objects": "always-multiline",
-          "imports": "always-multiline",
-          "exports": "always-multiline",
-          "functions": "never"
-        }
-      ],
-      "no-underscore-dangle": 0,
-      "no-param-reassign": 0,
-      "prefer-destructuring": 0,
-      "strict": 0
-    }
+    "test": "jest",
+    "posttest": "eslint .",
+    "webpack-defaults": "webpack-defaults",
+    "start": "npm run build -- -w",
+    "appveyor:test": "npm run test",
+    "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
+    "clean": "del-cli dist",
+    "lint": "eslint --cache src test",
+    "lint-staged": "lint-staged",
+    "prebuild": "npm run clean",
+    "prepublish": "npm run build",
+    "release": "standard-version",
+    "security": "nsp check",
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --collectCoverageFrom='src/**/*.js' --coverage",
+    "travis:lint": "npm run lint && npm run security",
+    "travis:test": "npm run test -- --runInBand",
+    "travis:coverage": "npm run test:coverage -- --runInBand"
   },
   "peerDependencies": {
-    "webpack": ">=0.9 <2 || ^2.1.0-beta || ^2.2.0"
+    "webpack": "^2.0.0 || >= 3.0.0-rc.0 || ^3.0.0"
   },
```

Comment:
Remove the `3.0.0-rc.0`. We don't support beta & RC builds once the major in question hits `@latest`